### DISCUSSION TITLE
Github maintenance?

### DISCUSSION THREAD

**Discussion by saewon1** [cooperrc/computational-mechanics#118] [General]
so I am trying to open the jypiter page and it says the uconn github servers are down for maintenance? and idea when they will be back online? I need to get project 3 done today

**Comment by dhb17001** [comment]
![unnamed](https://user-images.githubusercontent.com/112605381/196780402-58ae433b-8146-47af-8a65-7388bd117820.png)

I am also seeing this issue when trying to access the server with this error displayed above

**Comment by cooperrc** [comment]
It must have been some temporary maintenance on the github.uconn.edu site. I think its running now.
